src/Renderer/UI/ModelType.fs

```diff
@@ -287,13 +290,17 @@ type Model = {
     Tests: bool list
     /// Contains the thevenin parameters of a particular circuit
     TheveninParams: TheveninParameters option
+    /// Contains boolean for if the last clicked tab was properties or not
```

Comment:
Elmish state mutst be carefully considered (This has not always been done). It should be minimal. Values that can be calculated from otehr fields should be calculated - because this makes state update simpler.

Check whether any of these state values can be calculated and if so delete fields from state and add a method on model or a function of model that calculates them.